src/script/ScriptedItem.cpp

```diff
@@ -327,7 +327,46 @@ class SetPriceCommand : public Command {
 		
 		DebugScript(' ' << price);
 		
-		context.getEntity()->_itemdata->price = price;
+		context.getEntity()->_itemdata->buyPrice = price;
```

Comment:
would it make sense to have a modifier here so they arent both the same? where is the setprice command even used?

Reply:
This is the original command that I have left for backwards compatibility. It is used by the original game. I was thinking about adding an optional second parameter to the command instead of adding 2 more commands, but arx is terrible at handling optional parameters in scripts as can be seen in this previous PR: https://github.com/arx-insanity/ArxLibertatis/pull/48/files

Reply:
yeah im just wondering about how its used in scripts, so if it would be possible to just replace it with one of the new commands directly

Reply:
![image](https://github.com/arx-insanity/ArxLibertatis/assets/2386064/b7742d57-9618-45d4-9b73-5d41f3c90871)
That would require replacing every `set_price` which is like in 232 places inside 120 files